spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/ThreadPool.scala

```diff
@@ -107,7 +107,9 @@ class ThreadPool(private var poolSize: Int) {
 
       MKL.setNumThreads(size)
       BackendMklDnn.setNumThreads(size)
-      Affinity.setOmpAffinity()
+      if (System.getProperty("bigdl.disableOmpAffinity", "false").toBoolean) {
```

Comment:
why this change ? thought already in master?

Reply:
Yeah, I need to rebase master code.